packages/java-parser/src/comments.js

```diff
@@ -113,63 +28,153 @@ function findUpperBoundToken(tokens, comment) {
   return i;
 }
 
-/**
- * Extends each comments offsets to the left and the right in order to match the
- * previous and next token offset. This allow to directly match the prettier-ignore
- * comment to the correct CSTNode.
- * @param {*} tokens ordered array of tokens
- * @param {*} comments array of prettier-ignore comments
- * @return prettier-ignore comment array with extended location
- */
-function extendCommentRange(tokens, comments) {
-  const ignoreComments = [...comments];
+function isPrettierIgnoreComment(comment) {
+  return comment.image.match(
+    /(\/\/(\s*)prettier-ignore(\s*))|(\/\*(\s*)prettier-ignore(\s*)\*\/)/gm
```

Comment:
would /* prettier-ignore - blah blah */ be caught here?
should it be?

Reply:
No. We could support it, I guess, but as it is not supported in Prettier for JS, I think we should keep it like that for the moment